app/controllers/api/users_controller.rb

```diff
@@ -3,6 +3,49 @@ module Api
   class UsersController < ApplicationController
     include Devise::Controllers::Helpers
 
+    def index
+      @users = User.all
+
+      response = {
+        users: @users.map do |user|
+          {
+            id: user.id,
+            name: user.name,
+            email: user.email,
+          }
+        end
+      }
+
+      render json: response.to_json
+
+    end
+
+    def show
+      @user = User.find(params[:id])
+
+      puts params[:id]
+
+      scores = Score.all.order(played_at: :desc, id: :desc)
```

Comment:
Score.all is inefficient because it will get all the data from the db. Please use `Score.where(...).order...` instead